frontend/app/src/store/defi/protocols/index.ts

```diff
@@ -586,10 +586,7 @@ export const useDefiSupportedProtocolsStore = defineStore(
       const currentStatus = getStatus(section);
       const newStatus = refresh ? Status.REFRESHING : Status.LOADING;
 
-      if (
-        !isLoading(currentStatus) ||
-        (currentStatus === Status.LOADED && refresh)
-      ) {
+      if (!loading(section) || (currentStatus === Status.LOADED && refresh)) {
```

Comment:
I think it makes sense to move this check to a single function in the status updater composable 😃